spec/classes/foreman_proxy__proxydhcp__spec.rb

```diff
@@ -194,6 +194,39 @@
             'interfaces' => ['eth0']
         ) end
       end
+
+      context "as manager of ACLs for dhcp" do
+        let :facts do
+          facts.merge({:ipaddress_eth0 => '192.168.100.20',
```

Comment:
There's no need to override any facts in this testcase. In the above we override them because facterdb does not provide stable IPs, just some valid IP. When you're actually writing assertions based on that then you want to ensure they don't change and why we override them. In your testcase you can use the autodetection we have.

Reply:
Without these set, I was seeing the tests error out due to -- https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/proxydhcp.pp#L6

Reply:
Ah, I suppose those facts don't have the eth0 interface. It's ok to leave them in then.